openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-board.js

```diff
@@ -369,6 +370,52 @@ var Wb = function() {
 					minHeight: 140
 					, minWidth: 255
 				});
+				tools.find('.om-icon.dweet').click(function(){					
+			        dweet = OmUtil.tmpl("#wb-dweet");			        
```

Comment:
1. `OmUtil.tmpl` accepts new id as second parameter
2. please use single quotes for strings

Reply:
Done